### ISSUE TITLE
C# code incorrect for Logic Preferences page

### ISSUE THREAD

**Issue by somnivore** [godotengine/godot-docs#2195]
https://docs.godotengine.org/en/latest/getting_started/workflow/best_practices/logic_preferences.html

This page includes the following incorrect C# code:

```
using System;
using GD;

public class MyBuildings : public Node { 

    public const PackedScene Building { get; set; }

    public PackedScene ABuilding = GD.load("office.tscn") as PackedScene;

    public void MyBuildings() {
        // Can assign the value during initialization or during a constructor.
        Building = ResourceLoader.load("building.tscn") as PackedScene;

        // C# and other languages have no concept of "preloading".
    }
}
```

**Comment by somnivore** [comment]
There are more incorrect C# examples found in this commit: https://github.com/godotengine/godot-docs/commit/e8289b93f334c533890ae1e19cd345473ba2fac3#diff-2a9b7bff1587eb0e7db5f56d053ad8c0

**Comment by bojidar-bg** [comment]
Uhm, sorry. What is invalid in the example code posted above? (is it the `.load` as opposed to `.Load` or something more serious?)

**Comment by neikeq** [comment]
Another problem is the use of `const`. That should be readonly instead. I had a pending review assigned in a PR (not sure if the one that introduced this), but it got merged before I could review.
I will do a global review of the C# code this week, but it would be great if the code could be tested before commiting. You can use official builds with mono enabled, it's not necessary to build build from source.

**Comment by somnivore** [comment]
@bojidar-bg 
`using GD;` should be `using Godot;`, or at least `using Godot;` should be added somewhere, as otherwise Godot's classes won't be found.

`public class MyBuildings : public Node` is an incorrect way of extending a class, it should just be `: Node` without the `public`

`public const PackedScene Building { get; set; }` consts can't be used this way, they must be given a constant value in the declaration, and cannot use `{get; set;}` (given they're constant, after all, a setter would do nothing, and the getter wouldn't really be used for anything either.) As @neikeq mentions, `readonly` should be used instead (and optionally static, to mimic the way consts work.)

`public PackedScene ABuilding = GD.load("office.tscn") as PackedScene;` this line should use `.Load()`, and the path should be `"res://office.tscn"`

`public void MyBuildings()` this is an incorrect way of defining a constructor in C#, it should read `public MyBuildings()`

`Building = ResourceLoader.load("building.tscn") as PackedScene;` another case of using lowercase load, and an incorrect path.  Another user reported that the gdscript examples also used the incorrect path.

Also there was a recent addition of generic load, so it can be changed to `ResourceLoader.Load<PackedScene>()` instead (but this is preference I suppose.)

I think that's all of the problems with this page; other pages had peculiarities too, like `public SomeClass extends Node{`, seemingly mixing in gdscript's keywords into C#, which doesn't work either.

**Comment by neikeq** [comment]
These issues were fixed by #2196. I made other fixes in #2198.
